src/Views/HtmlViewBridge.js

```diff
@@ -773,7 +775,11 @@ HtmlViewBridge.prototype.raiseServerEvent = function (eventName) {
         argumentsArray[i] = arguments[i];
```

Comment:
Just read your note again - you mentioned server side. Yeah I presume those go through to the server as null or false or '' arguments? But yes, I was thinking of the raiseClientEvent bit below - so for both reasons it would be best to remove them from the arguments array